### ISSUE TITLE
intégrales multiples

### ISSUE THREAD

**Issue by plgitlogin** [PremierLangage/Yggdrasil#65]
URL de la fiche : https://docs.google.com/open?id=1T5Ykat0pzHB0OsgpxMbuqJi-O7CFdAfH3ex8eUP4nzw
Horodateur: 01/07/2019 14:56:53
Titre de l'exercice: intégrales multiples
Auteur: Claudine Degand 
Adresse mail: claudine.degand@u-pem.fr
Public visé: L2 de maths à l'UPEM - Ingénieurs en bac+3 
Contexte d’utilisation: à la maison - en TD 
Objectif pédagogique: Pratiquer des intégrales multiples - utilisation de Fubini - changement de variables 
Mots clés: Maths - outil mathématique - intégrales multiples
Description de l'exercice (texte): 
Description de l'exercice (Fichier): 
Champs de saisie: 
Valeurs aléatoires et algorithme qui les génère.: 
Comment détermine-t-on si la réponse est juste?: 
Feedback à l'étudiant: 
Fichier de données (tableau):

**Comment by nimdanor** [comment]
@esandier Fiche exo à améliorer c'est quoi l'amélioration a apporter ?

**Comment by esandier** [comment]
C'est un label qui indique que la fiche exo est pas définitive. En fait il faudrait un ensemble de labels qui indique l'état d'avancement. 'Fiche en cours de finalisation', 'Fiche OK', 'Codage en cours', et en même temps des labels "avertissement" destinés à un des intervenants comme "fiche à améliorer", "bug sur exercice codé", etc...
    Le lundi 1 juillet 2019 à 21:21:15 UTC+2, Dominique Revuz <notifications@github.com> a écrit :  
 
 
@esandier Fiche exo à améliorer c'est quoi l'amélioration a apporter ?

**Comment by nimdanor** [comment]
@esandier  je pense qu'il faut militer pour que si l'on ajoute ce tag il faut absolument faire un commentaire d'explication sur les améliorations a apporter. 
Le tag seul c'est triste comme : "c'est nul!"

**Comment by esandier** [comment]
Je suis d'accord mais je n'ai pas trouvé de formulation mieux. "Fiche en cours"?
    Le mardi 2 juillet 2019 à 10:31:52 UTC+2, Dominique Revuz <notifications@github.com> a écrit :  
 
 
@esandier je pense qu'il faut militer pour que si l'on ajoute ce tag il faut absolument faire un commentaire d'explication sur les améliorations a apporter.
Le tag seul c'est triste comme : "c'est nul!"
